README.md

```diff
@@ -20,7 +20,7 @@ Feature management provides a way to develop and expose application functionalit
 * [ASP.NET Core Web App (Razor Page)](./examples/RazorPages)
 * [ASP.NET Core Web App (MVC)](./examples/FeatureFlagDemo)
 * [Blazor Server App](./examples/BlazorServerApp)
-* [ASP.NET Core Web App with Feature Flag Telemetry](./examples/EvaluationDataToApplicationInsights)
+* [ASP.NET Core Web App with Variants and Telemetry](./examples/VariantAndTelemetryDemo)

```

Comment:
The example is called `VariantAndTelemetryDemo`. Can we use singular here for consistency if there is no grammer issue?

Reply:
Weirdly- I think the grammar makes sense. This is an app that has variants (plural). But when describing something as a demo- it doesn't need to be plural. Like FilterDemo (instead of FiltersDemo). 

If others feel strongly to align the two- I'd prefer changing the package to `VariantsAndTelemetryDemo`